src/io/RtInputScannerImages.cpp

```diff
@@ -367,6 +367,13 @@ RtExternalImageInfo *RtInputScannerImages::receiveImageInfo(
   // read until we have all the bytes we need
   // TODO add error handling here
 
+  // first 8 bytes are size of header, size of data
```

Comment:
since we know this is 8 bytes, should the sizes be explicitly set to 4 instead of `sizeof(int)`, which has varied over the years?